### ISSUE TITLE
[IMPROVEMENT] Allow user to switch between previous/next comment on timeline on scroll events

### ISSUE THREAD

**Issue by S-trace** [ris58h/youtube-timestamps#24]
Hello.

It can be handy if the user can switch between previous/next comments using scroll events (vertical scroll or horizontal scroll). because some videos can have many comments in a short timeframe, and it is difficult to target the next mark even on regular videos in the current implementation.

Thank you.

**Comment by ris58h** [comment]
Duplicate of #8

**Comment by S-trace** [comment]
This is not a duplicate of #8 

I mean not to change the playback position to the position marked in a comment, but to switch between comments on the timeline.
Comments may have very close markers, which require very precise mouse positioning to read the next comment and is annoying. Using the scroll wheel to show previous/next comment text (if the user hovers the mouse over one of the comments marks on the timeline) can make this extension better, I think.

**Comment by S-trace** [comment]
I have recorded a screencast for this issue. 
Please take a look at it - to view the next comment on the timeline I need to move my mouse very precisely, which is not comfortable. 

I have paused YouTube to show you only the effect I mentioned (and I had to reduce my mouse DPI to poison my mouse (I usually use 1000DPI precision, but reduced it to 400DPI for this,

https://user-images.githubusercontent.com/3058740/152663533-eeacfb65-44bb-4e35-a004-21d4f175c486.mp4

I think it may be useful to allow users to use mouse scroll events instead of cursor movement to switch previous/next comments (and even previous/next comments at all videos but not just around mouse pointer) without changing playback position.

**Comment by ris58h** [comment]
> Video can't be played because the file is corrupt.

Unfortunately I couldn't watch the video, but I understand the issue.

I played with a scroll events a little bit. I decided to use vertical scroll to scroll long comments (it's already in the `master` but haven't released yet; see https://github.com/ris58h/youtube-timestamps/commit/a0fac77fa17909940b38152d151ca54da2c34aa8). I'm not sure about UX of comments switching. Are you ok with horizontal scrolling for it?

UPD: version 0.5.2 is released.

**Comment by S-trace** [comment]
> Are you ok with horizontal scrolling for it?

Yes, using a horizontal scroll will be very intuitive in this context. 
Thank you!

**Comment by S-trace** [comment]
I'm sorry, it seems this bug is not only about comments with 0 likes but also is an internal bug of label positioning. 

Sometimes the comment label is displayed at the very beginning of the video:
![изображение](https://user-images.githubusercontent.com/3058740/156650823-41e68f98-4149-42d1-bb86-42c03a15465d.png)

But sometimes it is not displayed entirely:
![изображение](https://user-images.githubusercontent.com/3058740/156651536-6bc793a1-fa18-48a3-b164-ba7eb5def81a.png)

The comment:
![изображение](https://user-images.githubusercontent.com/3058740/156651627-dca1c705-27eb-4d47-9422-7eda6fe07059.png)

Video:
https://www.youtube.com/watch?v=mwCSEvTVwdQ&list=PLkdrA53ZXRAqPomwfdfLAujqdAupry0fP&index=110
